gst/nnstreamer/tensor_merge/gsttensormerge.c

```diff
@@ -761,26 +777,28 @@ gst_tensor_merge_change_state (GstElement * element, GstStateChange transition)
 /**
  * @brief Setup internal data (data_* in GstTensor_Merge)
  * @param[in/out] filter "this" pointer. mode & option MUST BE set already.
+ * @retval TRUE if ok. FALSE if given input is invalid.
  */
-static void
+static gboolean
 gst_tensor_merge_set_option_data (GstTensorMerge * filter)
 {
   if (filter->mode == GTT_END || filter->option == NULL)
-    return;
+    return TRUE;
```

Comment:
FALSE?

Reply:
If it is not configured (mode=GTT_END or option=NULL), we skip the operation.
I'll rephrase the comments of retval